.circleci/config.yml

```diff
@@ -67,6 +95,64 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
+  e2e-simple-cni:
+    <<: *integrationDefaults
+    environment:
+            - KUBECONFIG: /go/out/minikube.conf
+            - TEST_ENV: minikube-none
+            - GOPATH: /go
+    steps:
+      - <<: *initWorkingDir
+      - checkout
+      - attach_workspace:
+          at:  /go
+      - <<: *markJobStartsOnGCS
+      - run: make sync
+      - run: make build
+      - run:
+          command: |
+            if [ ! -f /go/out/linux_amd64/release/istio-cni ]; then
+              # Should only happen when re-running a job, and the workspace is gone
+              time make build
+            fi
+            make docker.all
+      - run: docker images
+      - run:
+          name: Get Istio
```

Comment:
I had started a small sh script to do this.  It was in a different branch that i haven't submitted a PR for.  The reason I started this with a shell script is so that if someone wanted to run the tests locally it would be done in a uniform way.   Although re-considering since there is some complexity to doing it in a script and instead we can require the user checkout and build Istio anyway they want.

Reply:
" bin/testEnvRootMinikube.sh -- add arg for CNI"   what arg are you thinking is needed?  When I went through the script I thought it could be used as is?   
   
    "pull in patch for making e2e_simple use istioctl kubeinject with configmap for sidecar inject to not inject initContainer"

That patch merged so latest master would have it. Also Steves patch is needed (which also merged).  Finally and most importantly your patch is needed which I don't think there is an open PR yet for.

Reply:
RE: testEnvRootMinikube.sh -- minikube doesn't configure kubelet to use CNI by default.  It has a command line param that it doesn't look like that script sets when starting minikube.

RE: patches -- Right... that's why I used my private repo.